convene-web/app/controllers/features_controller.rb

```diff
@@ -0,0 +1,17 @@
+class FeaturesController < ApplicationController
+  def index
+
+  end
+
+  def show
+
+  end
+
+  helper_method def feature
+    @feature ||= Feature.find_by!(id: params[:id].to_sym)
+  end
+
+  helper_method def features
+    @features ||= Feature.all
+  end
```

Comment:
I tend to prefer to use [decent_exposure](https://github.com/hashrocket/decent_exposure) for exposing models to the view; as it helps keep controllers light; but I am content to wait a bit.